.travis.yml

```diff
@@ -1,6 +1,7 @@
 language: go
 go:
-- 1.6.3
```

Comment:
If we actually wanted to get this merged in to rollrus, would we need to support this?

Reply:
probably. It would be up to them. 1.6 is getting pretty old